app/navigation/nonsipp/NonSippNavigator.scala

```diff
@@ -60,11 +67,14 @@ class NonSippNavigator @Inject()() extends Navigator {
         } else {
           routes.HowMuchCashController.onPageLoad(srn, NormalMode)
         }
+
+      case PensionPaymentsReceivedPage(_) => controllers.routes.UnauthorisedController.onPageLoad()
```

Comment:
We have individual navigators for all the sub journeys now so create a new one and add it to the `journeys` value below